op-node/rollup/engine/engine_kind.go

```diff
@@ -0,0 +1,44 @@
+package engine
+
+import "fmt"
+
+// EngineClientKind identifies the engine client's kind, used to control the behavior of optimism in different engine clients.
+type EngineClientKind string
+
+const (
+	EngineClientGeth   EngineClientKind = "geth"
+	EngineClientReth   EngineClientKind = "reth"
+	EngineClientErigon EngineClientKind = "erigon"
```

Comment:
I propose to shorten all occurrences of `EngineClient` to `Engine` everywhere, e.g.
```suggestion
type EngineKind string

const (
	EngineGeth   EngineKind = "geth"
	EngineReth   EngineKind = "reth"
	EngineErigon EngineKind = "erigon"
```

I don't think that omitting the `Client` leads to any ambiguity.

Reply:
Or seeing that they are in package `engine`, maybe even drop the `Engine`. That would be idiomatic in Go, so they can be used as `engine.Kind`, `engine.Geth` etc.

Reply:
I agree that it looks much better! I'll modify it to Engine.Kind / Engine.Geth.